Makefile

```diff
@@ -35,10 +42,10 @@ deploy-cert-manager:
 		--set installCRDs=true \
 		--set extraArgs[0]="--enable-certificate-owner-ref=true"
 
-%-konk-operator: HELM_FLAGS ?= --set=image.tag=$(GIT_VERSION) --set=image.pullPolicy=IfNotPresent
+%-konk-operator: HELM_FLAGS ?= --set=image.pullPolicy=IfNotPresent
 
 deploy-%: package
-	$(HELM) upgrade -i --wait $(RELEASE_NAME)-$* $(CHART_DIR)/$* $(HELM_FLAGS)
+	$(HELM) upgrade -i --wait $(RELEASE_NAME)-$* $(CHART_DIR)/$* --set=image.tag=$(GIT_VERSION) $(HELM_FLAGS)
```

Comment:
It's sloppy to set `image.tag` on `make deploy-konk`, even if that value is currently unused in that chart. That is why the flag previously only applied to `*-konk-operator`.

Reply:
not sure I am following. konk and konk-operator would both have the same version from git

Reply:
konk's image tag does not track the git version; it tracks k8s/apiserver/etcd versions. So you don't want to be overriding `image.tag` on konk with the git version.